.gitignore

```diff
@@ -65,4 +65,4 @@ test-results/
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
-!.yarn/versions
+.yarn/versions
```

Comment:
For my understanding, can you explain this change?

Reply:
When we bump version in this way there'll be fine generated locally under `.yarn/versions/` and each file is empty. That's why i ignored this folder. any other suggestions?

Reply:
According to the [the docs](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored), we might want to commit these files with the repo. Can you confirm this with @Gudahtt?

Reply:
thanks for sharing the docs, was not aware of that config. now reverted back 🙏